includes/classes/pattern/class-flexible-header.php

```diff
@@ -266,10 +266,11 @@ function get_pip_header( $echo = true ) {
 
     $pip_flexible_header = acf_get_instance( 'PIP_Flexible_Header' );
     $pip_pattern         = acf_get_instance( 'PIP_Pattern' );
+    $pip_pattern_id      = acf_get_valid_post_id( $pip_pattern->pattern_post_id );
 
     if ( $echo ) {
-        echo get_flexible( $pip_flexible_header->get_flexible_header_field_name(), $pip_pattern->pattern_post_id );
+        echo get_flexible( $pip_flexible_header->get_flexible_header_field_name(), $pip_pattern_id );
     } else {
-        return get_flexible( $pip_flexible_header->get_flexible_header_field_name(), $pip_pattern->pattern_post_id );
+        return get_flexible( $pip_flexible_header->get_flexible_header_field_name(), $pip_pattern_id );
```

Comment:
I wonder if it's not better to change directly in the class the correct post_id:
https://github.com/Pilot-in/PiloPress/blob/b3d2fd47f68edf8bc53ddc80b96514ab9899d40d/includes/classes/pattern/class-pattern.php#L24-L32

into:
```php
/**
* Post ID
*
* @var string
*/
public $pattern_post_id = 'pip_pattern';

public function __construct() {

    $this->pattern_post_id = acf_get_valid_post_id( $this->pattern_post_id );
```